docs/smart-contracts/entrypoints.md

```diff
@@ -16,6 +16,7 @@ Unlike functions and API endpoints, entrypoints do not return a value directly t
 To return a value from a smart contract, you can use one of these methods:
```

Comment:
This could be misleading, as onchain `Views` is indeed a way for returning value from contract to contract. But the `Events` is not. It's an indirect way, as an off-chain routine is needed for triggering following up actions.

Reply:
Changed this to:
- Use Views to return data to smart contracts or off-chain applications
- Use Events to return data to off-chain applications